### DISCUSSION TITLE
绳网什么时候关啊

### DISCUSSION THREAD

**Discussion by Phaethon0** [share121/inter-knot#5729] [公告]
现在只能用投票发帖子了么？

**Comment by c328422** [comment]
可能是因為發文的連結掛了？畢竟現在基本無人維護的狀態，之前up就說要關閉，可能現在是回光返照而已

**Comment by Rorschachac** [comment]
投票暂不支持是我设置的问题吗
![367565134-792eff50-f10c-47da-a59b-e79b4a77254c](https://github.com/user-attachments/assets/ba723261-f0ab-4d32-b9e4-be550f6719a4)

**Reply by Rexia-404** [reply]
好像我也不行，好遗憾哦……
